### PR TITLE
Quiz and Clickventure embeds resize (Kinja) parent frame

### PR DESCRIPTION
**Pull request by mparent61**
Used for Clickhole migration to Kinja.

Leverages Kinja Mantle's built-in support for passing resize messages from within an embed iframe.

### CODE REVIEW

**Comment by mparent61** [comment]
@spra85 -- still need to test via temp release, but is this what you had in mind?

File: elements/bulbs-clickventure/bulbs-clickventure.js [hunk 1]
```diff
@@ -18,6 +19,9 @@ class BulbsClickventure extends BulbsHTMLElement {
     let $element = $(this);
     $element.data('clickventurePlugin', new Clickventure($element));
     $element.on('clickventure-page-change', fireAnalytics);
+
+    // Embeded CVs can resize their parent frame
+    resizeParentFrame(document.getElementsByTagName('body')[0].clientHeight);
```
Comment (collin) [inline elements/bulbs-clickventure/bulbs-clickventure.js]: Looks nice, can use the shortcut `document.body` here to get the body element.

Comment (mparent61) [reply elements/bulbs-clickventure/bulbs-clickventure.js]: Thank you kind sir!

File: lib/bulbs-elements/util/resize-parent-frame.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
+// Resize a (Kinja) parent frame, using Kinja's parent notification support.
+
+export default function resizeParentFrame (height) {
+  if (window.parent !== null) {
```
Comment (spra85) [inline lib/bulbs-elements/util/resize-parent-frame.js]: Should be able to just use: 

```
if (window.parent) {
```

**Pull request merged**